[2024-05-28T15:33:42.362+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-05-28T15:33:42.386+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: stored_procedures.load_reference_table manual__2024-05-28T15:33:30.069104+00:00 [queued]>
[2024-05-28T15:33:42.396+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: stored_procedures.load_reference_table manual__2024-05-28T15:33:30.069104+00:00 [queued]>
[2024-05-28T15:33:42.397+0000] {taskinstance.py:2303} INFO - Starting attempt 1 of 2
[2024-05-28T15:33:42.411+0000] {taskinstance.py:2327} INFO - Executing <Task(PythonOperator): load_reference_table> on 2024-05-28 15:33:30.069104+00:00
[2024-05-28T15:33:42.418+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=1475) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-05-28T15:33:42.421+0000] {standard_task_runner.py:63} INFO - Started process 1477 to run task
[2024-05-28T15:33:42.420+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'stored_procedures', 'load_reference_table', 'manual__2024-05-28T15:33:30.069104+00:00', '--job-id', '138', '--raw', '--subdir', 'DAGS_FOLDER/dag_example.py', '--cfg-path', '/tmp/tmpynjxw2wv']
[2024-05-28T15:33:42.422+0000] {standard_task_runner.py:91} INFO - Job 138: Subtask load_reference_table
[2024-05-28T15:33:42.473+0000] {task_command.py:426} INFO - Running <TaskInstance: stored_procedures.load_reference_table manual__2024-05-28T15:33:30.069104+00:00 [running]> on host 29c09f03cc50
[2024-05-28T15:33:42.554+0000] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Yamin' AIRFLOW_CTX_DAG_ID='stored_procedures' AIRFLOW_CTX_TASK_ID='load_reference_table' AIRFLOW_CTX_EXECUTION_DATE='2024-05-28T15:33:30.069104+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-05-28T15:33:30.069104+00:00'
[2024-05-28T15:33:42.556+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-05-28T15:33:42.576+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-05-28T15:33:42.577+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-05-28T15:33:42.587+0000] {taskinstance.py:1205} INFO - Marking task as SUCCESS. dag_id=stored_procedures, task_id=load_reference_table, execution_date=20240528T153330, start_date=20240528T153342, end_date=20240528T153342
[2024-05-28T15:33:42.636+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-05-28T15:33:42.663+0000] {taskinstance.py:3482} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-05-28T15:33:42.664+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
